index.html

```diff
@@ -86,25 +88,23 @@
       <div class="grid gap-10 mb-10 mt-10 xl:mt-0">
         <!-- section one for real estate -->
         <section
-          class="px-3 xl:px-0 bg-color-all flex flex-col gap-12 xl:gap-0"
+          class="px-3 xl:px-0 bg-[#141414] flex flex-col gap-12 xl:gap-0"
```

Comment:
this bg is used at so many places, why not add this color to tailwind config?

Reply:
also set in config file